src/OrleansRuntime/Async/CancellationSourcesExtension.cs

```diff
@@ -14,11 +14,11 @@ internal class CancellationSourcesExtension : ICancellationSourcesExtension
     {
         private readonly Lazy<TraceLogger> _logger = new Lazy<TraceLogger>(() =>
             TraceLogger.GetLogger("CancellationSourcesExtension", TraceLogger.LoggerType.Application));
-        private static readonly Interner<Guid, GrainCancellationToken> _cancellationTokens;
-        private static readonly TimeSpan _cleanupFrequency = TimeSpan.FromMinutes(3);
-        private static readonly int _defaultInternerCollectionSize = 31;
+        private readonly Interner<Guid, GrainCancellationToken> _cancellationTokens;
+        private readonly TimeSpan _cleanupFrequency = TimeSpan.FromMinutes(3);
```

Comment:
_cleanupFrequency and _defaultInternerCollectionSize are constant, so definitely need to be static. there is just no need to have them per instance.